react-frontend/src/components/CaseStudies/casestudies.js

```diff
@@ -1,18 +1,37 @@
 import React from 'react';
 import CaseStudiesBanner from './casestudiesbanner';
 import CardList from './cardlist';
+import { Switch, Route, useRouteMatch } from 'react-router-dom';
+import Template from './template';
+import { MiningContent, FarmerContent, MedicalContent } from './content';
 
 import '../../css/casestudies.css';
 
-function CaseStudies() {
+const CaseStudies = () => {
+  let { path, url } = useRouteMatch();
   return (
-    <div className="caseStudyContainer">
-      <CaseStudiesBanner />
-      <div className="caseStudyBody">
-        <CardList />
-      </div>
+    <div>
+      <Switch>
+        <Route exact path={path}>
+          <div className="caseStudyContainer">
+            <CaseStudiesBanner />
+            <div className="caseStudyBody">
+              <CardList />
+            </div>
+          </div>
+        </Route>
+        <Route path={path + '/mining-study'}>
```

Comment:
https://medium.com/better-programming/how-to-pass-multiple-route-parameters-in-a-react-url-path-4b919de0abbe#:~:text=dynamic%20content%20required.-,Route%20parameters,data%20based%20on%20the%20parameter.

Reply:
just a reference to a more DRY approach!